manager/src/test/java/com/redhat/service/bridge/manager/dao/ProcessorDAOTest.java

```diff
@@ -176,7 +177,7 @@ public void findByBridgeIdAndCustomerId_pageOffset() {
         Processor p = createProcessor(b, "foo");
         Processor p1 = createProcessor(b, "bar");
 
-        ListResult<Processor> listResult = processorDAO.findByBridgeIdAndCustomerId(b.getId(), TestConstants.DEFAULT_CUSTOMER_ID, 1, 1);
+        ListResult<Processor> listResult = processorDAO.findByBridgeIdAndCustomerId(b.getId(), TestConstants.DEFAULT_CUSTOMER_ID, new QueryInfo(1, 1));
         assertThat(listResult.getPage()).isEqualTo(1L);
```

Comment:
Same as above. I would use PAGE_ONE and PAGE_SIZE_ONE here. Besides you can reuse these constants in the following assertions, and make the binding between test setup and test assertions clearer.

Also you may want to consider using DEFAULT_CUSTOMER_ID as a static import, the TestConstants. prefix is just noise in this test.